test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -5179,7 +5178,7 @@ def revert_to_defaults(self, reverthooks=True, revertqueues=True,
             except:
                 rescs = []
             if len(rescs) > 0:
-                self.manager(MGR_CMD_DELETE, RSC, id=rescs, expect=True)
```

Comment:
Shouldn't we need to add expext() after manager() since we removed expect=True?

Reply:
I can, but the logic behind removing expect=True was that it was not necessary.  The manager() call itself will raise an exception if it fails.  We should trust PBS to do the operation properly if it didn't fail.